src/site/xdoc/server/config-blob-export.xml

```diff
@@ -50,6 +51,24 @@
               </dd>
           </dl>
       </subsection>
+      <subsection name="LinShare Blob Export Configuration">
```

Comment:
Again in this file adding a little comment about what file export is, which features it is currently used for, and the fact that it is only applicable with guice would really help understanding what it is used for.